frontend/model/contracts/chatroom.js

```diff
@@ -511,7 +511,7 @@ sbp('chelonia/defineContract', {
             shouldDeleteToken: me === data.messageSender
           }
           deleteEncryptedFiles(data.manifestCids, option).catch(e => {
-            console.error(`[gi.contracts/chatroom/deleteMessage/sideEffect] (${contractID}):`, e)
+            console.warn(`[gi.contracts/chatroom/deleteMessage/sideEffect] (${contractID}):`, e)
```

Comment:
Is there a way to keep this as an error (as it's important for users to know if the file failed to delete), by modifying the code above to detect if a file has already been deleted (and not throwing if that's the case?)